src/core/Akka/Util/Internal/TaskEx.cs

```diff
@@ -31,14 +28,7 @@ internal static class TaskEx
         /// </summary>
         public static TaskCompletionSource<T> NonBlockingTaskCompletionSource<T>()
         {
-            if (IsRunContinuationsAsynchronouslyAvailable)
-            {
-                return new TaskCompletionSource<T>((TaskCreationOptions)RunContinuationsAsynchronously);
-            }
-            else
-            {
-                return new TaskCompletionSource<T>();
-            }
+            return new TaskCompletionSource<T>(TaskCreationOptions.RunContinuationsAsynchronously);
```

Comment:
All TCS instances can now run with `TaskCreationOptions.RunContinuationsAsynchronously` by default in .NET Standard 2.0+